sources/kube.go

```diff
@@ -259,6 +272,7 @@ func newKubeSource() (*KubeSource, error) {
 		client:      kubeClient,
 		lastQuery:   time.Now(),
 		kubeletPort: *argKubeletPort,
+		activePods:  make(map[PodInstance]bool),
```

Comment:
This might result in linear memory growth since we don't prune the activePods map at all.